virttest/utils_test/qemu/__init__.py

```diff
@@ -610,7 +610,9 @@ def hotplug_memory(self, vm, name):
                 dev_type = "pc-dimm"
             step = "Hotplug %s '%s' to VM" % (dev_type, dev.get_qid())
             error_context.context(step, logging.info)
-            vm.devices.simple_hotplug(dev, vm.monitor)
+            _, ver_out = vm.devices.simple_hotplug(dev, vm.monitor)
+            if ver_out is False:
+                raise exceptions.TestError("Verify hotplug memory failed")
```

Comment:
`exceptions.TestFail` is more appropriate.

Reply:
@yanan-fu You are right, updated, thanks!